src/services/firebase/firestore.ts

```diff
@@ -221,18 +221,18 @@ export class FirestoreServices {
   };
 
   sendMessageWithFile = async (message: SendMessageProps) => {
-    const { path, extension, type } = message;
+    const { path, extension, type, fileName, size } = message;
 
-    if (!path || !extension || this.conversationId === null) {
-      console.error('Please provide path and extension');
-      return;
+    if (!path || !extension || this.conversationId === null || !type) {
+      throw new Error('Please provide path and extension');
     }
 
     try {
       const uploadResult = await uploadFileToFirebase(
```

Comment:
when sending large file which requires a bit long time to uploading, user will not see the chat appearing on screen. I think we should implement some loading indicator here in that case

Reply:
we will have a task in the future to handle indicator with document and video, audio

Reply:
ok I agree